src/com/stanimiriliev/chaptertwo/homogeneoustree/Tree.java

```diff
@@ -1,56 +1,50 @@
 package com.stanimiriliev.chaptertwo.homogeneoustree;
 
 /**
- * Binary tree
- * Created by Stanimir Iliev on 18.6.2017 y..
+ * Binary Ordered Tree, Homogeneous Tree
+ * @author  Stanimir
+ * @since 18.6.2017
  */
 public class Tree {
```

Comment:
Ok, I have a question.

`public class Tree {

    private Node root;

    public Tree(int root) {
        this.root = new Node(root);
    }

    public boolean contains(int element) {
        if(element == this.root.value)
            return true;
        else if(element < this.root.value) {
            if(this.root.left != null)
                return this.root.left.contains(element);//################################
            else
                return false;
        }
        else {
            if(this.root.right != null)
                return this.root.right.contains(element);//###############################
            else
                return false;
        }
    }

    class Node {
        public final int value;
        public Node left;
        public Node right;

        public Node(int value){
            this.value = value;
            this.left = this.right = null;
        }
    }
}`

Why on the marked places (''//###') the program cannot find the method 'contains'?